src/common/hooks/useReactSettings.ts

```diff
@@ -67,6 +68,7 @@ export function useReactSettings() {
   const settings: ReactSettings = {
     show_pdf_preview: true,
     react_notification_link: true,
+    show_document_preview: true,
```

Comment:
We definitely want this _false_ by default.

Reply:
Sure, changed!